include/mbedtls/config_psa.h

```diff
@@ -848,6 +843,11 @@ extern "C" {
 
 #endif /* MBEDTLS_PSA_CRYPTO_CONFIG */
 
+#if defined(PSA_WANT_ALG_ECDSA) && defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR) && \
+    defined(PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY)
+#define PSA_HAVE_FULL_ECDSA 1
+#endif
+
```

Comment:
I'm OK with moving this code forward in the file, but I was wondering: is it possible to have `PSA_WANT` symbols when `MBEDTLS_PSA_CRYPTO_CONFIG` is not defined? 
I mean, without it we don't include `crypto_config.h` and also most of the `config_psa.h` is also skipped (at least the part which automatically sets `PSA_WANT` symbols).

Reply:
We do have `PSA_WANT` whether `MBEDTLS_PSA_CRYPTO_CONFIG` is defined or not. The difference that `MBEDTLS_PSA_CRYPTO_CONFIG` makes is whether `PSA_WANT` symbols are calculated from `MBEDTLS` symbols or the other way round.

(I don't know how this applies specifically here.)